lib/commonTypes.ts

```diff
@@ -33,3 +35,52 @@ export interface GetDaoProposalsConfig {
    */
   proposalId?: Hash;
 }
+
+export enum SchemePermissions {
```

Comment:
why again do we have to duplicate this? why cant index.d.ts import and export these?

Reply:
We're just in an in-between stage right now.  The goal is to automatically generate index.d.ts, by whatever means.  We're just not there yet.  But getting closer...

Reply:
even now though you could import/export. not a big deal, just wondering why not

Reply:
1) I only just recently finished converting the code to typescript and there has been a bunch of higher priority stuff to work on.
2) You may be right that it is easy to do, but I've found no evidence that it is as easy as you say, I've been assuming it is not.
3) So I would like to be able to concentrate on that task and not mix it in with other stuff.